### PR TITLE
Integrated Mongoose, connected to MongoDB Atlas

### PR DESCRIPTION
**Pull request by donjewey**
#3 Added a database configuration file to connect to the mongoDB URI specified in `.env`. Also added a boilerplate Users model.

### CODE REVIEW

**Review by milos381 — APPROVED** [review]

**Review by milos381 — CHANGES_REQUESTED** [review]
withdrawing approval due to travis

**Review by milos381 — APPROVED** [review]

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "extends": ["eslint:recommended", "airbnb-base"],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest"
+    },
+    "plugins": ["@typescript-eslint"],
+    "rules": {
+        "indent": [
+            "warn",
+            2,
```
Comment (shalarewicz) [inline .eslintrc.json:16]: Change to `indent` 4 and `SwitchCase` to 2 to avoid conflict with prettier

File: build/b88d04fba731603756b1.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// extracted by mini-css-extract-plugin
```
Comment (shalarewicz) [inline build/b88d04fba731603756b1.scss:1]: We should probably add the build folder to `.gitignore`

File: server/models/User.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  handle: {type: String, required: true, unique: true},
```
Comment (shalarewicz) [inline server/models/User.ts:4]: Let's remove things we don't need from the schema (handle, bio, listings). 

See issue #3 for the query information that needs to be tracked in the database for now.

**Review by shalarewicz — CHANGES_REQUESTED** [review]

File: build/b88d04fba731603756b1.scss (hunk 2 above)
Comment (shalarewicz) [reply build/b88d04fba731603756b1.scss:1]: This is still outstanding. Lets first merge #6 and then this PR to get the updated config. Can update the .gitignore at that time

Comment (donjewey) [reply build/b88d04fba731603756b1.scss]: added build to gitignore

**Pull request merged**